shared/components/Choice/Choice.modules.scss

```diff
@@ -24,7 +24,7 @@
   justify-content: center;
   cursor: pointer;
 
-  margin-top: 4px;
+  margin-top: 3px;
```

Comment:
I'd call this a "magic number", meanings its a very random looking number that just happens to work. But why? Why 3 and not 4? Is 3 based on the font size? Is it exactly some percentage of the font size or line height?

Magic numbers are often very confusing and forces us to pixel push all the time to do adjustments.

I suggest taking a little bit of time to see if this can be based on an established variable that we have, or a percentage, or something.

If it is necessary to have a magic number here, lets at least have a comment explaining why its there and what it means.